spec/rspec/expectations/configuration_spec.rb

```diff
@@ -47,7 +45,15 @@ class << rspec_dup; undef configuration; end
       end
 
       context 'on an interpreter that does not provide BasicObject', :uses_should, :unless => defined?(::BasicObject) do
-        before { RSpec::Expectations::Syntax.disable_should(Delegator) }
+        include RSpec::Support::InSubProcess
```

Comment:
This line shows up again and again in this PR.  WDYT of `RSpec.configure { |c| c.include RSpec::Support::InSubProcess }` in `spec_helper`?